app-text/pdfgrep/pdfgrep-2.1.2.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+KEYWORDS="~amd64 ~x86"
+SRC_URI="https://www.pdfgrep.org/download/${P}.tar.gz"
+
+DESCRIPTION="A tool similar to grep which searches text in PDFs"
+HOMEPAGE="https://pdfgrep.org/"
+
+SLOT="0"
+LICENSE="GPL-2"
+IUSE="+pcre test unac"
+
+RDEPEND="
+	app-text/poppler:=[cxx]
+	dev-libs/libgcrypt:0=
+	pcre? ( dev-libs/libpcre[cxx] )
+	unac? ( app-text/unac )"
+BDEPEND="
+	${RDEPEND}
```

Comment:
`RDEPEND` does not go into `BDEPEND`. Instead, restore `DEPEND="${RDEPEND}"`

Reply:
Fixed, thanks!